frontend/components/Auction.vue

```diff
@@ -305,7 +317,10 @@ export default Vue.extend({
     computed: {
         errorText(): string | null {
             if (!this.isAuctionsLoading && !this.auction) {
-                this.$emit('fetchFinishedAuction', this.auctionId);
+                this.$emit('fetchTakeEventsFromAuction', this.auctionId);
+                if (this.takeEvents) {
+                    return 'This auction is finished';
+                }
                 return 'This auction was not found';
```

Comment:
Does that mean in the moment when there is no `this.takeEvents` yet, `This auction was not found` will be displayed?

Reply:
Currently during all my tests, the loading animation is set until all the events have been retrieved. Therefore `This auction was not found` was not been displayed to me in my testing. 

However I understand where you are coming from. Would it make sense to introduce a new prop called `eventsLoading` and to check that?

Reply:
> Currently during all my tests, the loading animation is set until all the events have been retrieved

This is not in any way confirmed by the code I see. `isAuctionsLoading` is only set inside the action querying auctions on the page load. It is not set inside events querying. If you testing it on the hardhat I imaging the loading times are very short to notice it. `isEventsLoading` would be ok, a better solution would be `isAuctionLoadingStore: Record<string, boolean>`